client/components/product-expiration/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Moment {
+	isValid: () => boolean;
+	format: ( arg0: string ) => string;
+	diff: ( arg0: Date ) => number;
```

Comment:
Should `arg0` be called `date` or something?

Also, I think that in practice `diff()` can work with `Moment|String|Number|Date|Array` and not just `Date`.

Reply:
Resolved in 16feb54d14598303f399cc90336be44c634d157b

Reply:
Re: item types, yes, but I'm just looking for an interface this component uses, not _all_ components. And `diff` here is hard-coded [`new Date()`](https://github.com/Automattic/wp-calypso/pull/38486/files#diff-85a7307dfce296093673e95c6e2be6f8R56).